lib/va_profile/profile_information/service.rb

```diff
@@ -0,0 +1,201 @@
+# frozen_string_literal: true
+
+require 'common/client/concerns/monitoring'
+require 'common/client/errors'
+require 'va_profile/service'
+require 'va_profile/stats'
+require_relative 'configuration'
+require_relative 'transaction_response'
+require_relative 'person_response'
+
+module VAProfile
+  module ProfileInformation
+    class Service < VAProfile::Service
+      include Common::Client::Concerns::Monitoring
+
+      STATSD_KEY_PREFIX = "#{VAProfile::Service::STATSD_KEY_PREFIX}.profile_information".freeze
+      configuration VAProfile::ProfileInformation::Configuration
+
+      OID = MPI::Constants::VA_ROOT_OID
+      AAID = '^NI^200DOD^USDOD'
+
+      CONTACT_INFO_CHANGE_TEMPLATE = Settings.vanotify.services.va_gov.template_id.contact_info_change
+      EMAIL_PERSONALISATIONS = {
+        address: 'Address',
+        residence_address: 'Home address',
+        correspondence_address: 'Mailing address',
+        email: 'Email address',
+        phone: 'Phone number',
+        home_phone: 'Home phone number',
+        mobile_phone: 'Mobile phone number',
+        work_phone: 'Work phone number'
+      }.freeze
+
+      # def get_response(type)
+      #   with_monitoring do
+      #     icn_with_aaid_present!
+      #     model = "VAProfile::Models::#{type.capitalize}".constantize
+      #     raw_response = perform(:post, path, { bios: [{ bioPath: 'bio' }] })
+      #     response = model.response_class(raw_response)
+      #     Sentry.set_extras(response.debug_data) unless response.ok?
+      #     response
+      #   end
+      # rescue Common::Client::Errors::ClientError => e
+      #   if e.status == 404
+      #     log_exception_to_sentry(
+      #       e,
+      #       { vet360_id: @user.vet360_id },
+      #       { va_profile: :person_not_found },
+      #       :warning
+      #     )
+      #     return PersonResponse.new(404, person: nil)
+
+      #   elsif e.status >= 400 && e.status < 500
+      #     return PersonResponse.new(e.status, person: nil)
+      #   end
+      #   handle_error(e)
+      # rescue => e
+      #   handle_error(e)
+      # end
+
+      def get_person
+        with_monitoring do
+          vet360_id_present!
+          raw_response = perform(:get, @user.vet360_id)
+
+          PersonResponse.from(raw_response)
+        end
+      rescue Common::Client::Errors::ClientError => e
+        if e.status == 404
+          log_exception_to_sentry(
+            e,
+            { vet360_id: @user.vet360_id },
+            { va_profile: :person_not_found },
+            :warning
+          )
+
+          return PersonResponse.new(404, person: nil)
+        elsif e.status >= 400 && e.status < 500
+          return PersonResponse.new(e.status, person: nil)
+        end
+
+        handle_error(e)
+      rescue => e
+        handle_error(e)
+      end
+
+      def self.get_person(vet360_id)
```

Comment:
Does the initialize method for this class's constructor accepts an object with arbitrary attributes (the stub user struct)? It seems like [it expects a user on the base class](https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/va_profile/service.rb#L15-L17) unless I'm missing something

Reply:
This will be removed in when merging the other services.